test/concurrent/ThreadTest.ooc

```diff
@@ -45,15 +45,17 @@ ThreadTest: class extends Fixture {
 				value set(value get() + 1)
 				Thread yield()
 				Time sleepMilli(1)
+				if (Thread currentThread() isCanceled())
```

Comment:
@marcusnaslund I don't think it is complex, as it requires to introduce one class, if we make the base ThreadTask not abstract and not cancellable. C++ has no interfaces too and such approach would work just fine. I don't care if it is an interface or something else, I just want it to be modular and flexible.